tests/api.c

```diff
@@ -41422,7 +41422,218 @@ static int test_wolfSSL_SESSION(void)
     return EXPECT_RESULT();
 }
 
-#if defined(OPENSSL_EXTRA) && defined(HAVE_SSL_MEMIO_TESTS_DEPENDENCIES) && \
+#if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && \
+    !defined(NO_RSA) && defined(HAVE_IO_TESTS_DEPENDENCIES) && \
+    !defined(NO_SESSION_CACHE) && defined(OPENSSL_EXTRA) && \
+    !defined(WOLFSSL_NO_TLS12)
+static void set_ctx_timeout(WOLFSSL_CTX* ctx)
```

Comment:
This function name needs to be namespaced with a prefix. I usually go for `test_wolfSSL_SESSION_expire_downgrade_ctx_ready`.